src/AbstractActionHandler.ts

```diff
@@ -127,6 +140,11 @@ export abstract class AbstractActionHandler {
    */
   protected abstract async handleWithState(handle: (state: any, context?: any) => void): Promise<void>
 
+  /**
+   * Checks that the required setup has occurred.
+   */
+  protected abstract async isSetup(): Promise<boolean>
```

Comment:
I realize now that, for the same reason the error should be named this way, this should really be `isSetUp`.